packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -139,6 +139,29 @@ function createCompiler(webpack, config, appName, urls, useYarn) {
 
   let isFirstCompile = true;
 
+  compiler.plugin('after-compile', (compilation, callback) => {
+    // Order compilation warnings by most recently modified
+    compilation.warnings = [
+      ...compilation.warnings,
+    ].sort((warning1, warning2) => {
+      if (!warning1._lastModifiedDate) {
+        warning1._lastModifiedDate = fs.statSync(
```

Comment:
Is there a way to avoid these calls every time? I think webpack may expose this information in a cached way via its own "filesystem" object. Or maybe there's some other global object that already contains last edit times. Could you check for this?